### PR TITLE
[1LP][RFR] Host.execute_button() converted to widgetastic

### PR DESCRIPTION
**Pull request by quarckster**
Purpose
=================
`execute_button` method of Host object converted to widgetastic.

{{pytest: -v -k "test_button_on_host"}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 627d7e921d8852689cad48c5fa6dbee43f173b43:

`cfme/common/host_views.py`:
- :red_circle: Line 2:1: **F401** *'widgetastic.utils.ParametrizedString' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/infrastructure/host.py [hunk 1]
```diff
@@ -189,22 +189,9 @@ def load_details(self, refresh=False):
...
-        host_btn = partial(tb.select, button_group)
-        host_btn(button, invokes_alert=True)
-        sel.click(form_buttons.submit)
-        flash.assert_success_message("Order Request was Submitted")
-        host_btn(button, invokes_alert=True)
-        sel.click(form_buttons.cancel)
-        flash.assert_success_message("Service Order was cancelled by the user")
+    def execute_button(self, button_group, button, handle_alert=False):
+        view = navigate_to(self, "Details")
+        view.toolbar.custom_button(button_group).item_select(button, handle_alert=handle_alert)
```
Comment (izapolsk) [inline cfme/infrastructure/host.py]: your implementation of execute_button doesn't check flash messages. Why ?

**Comment by izapolsk** [comment]
PRT failed. can you fix test or explain errors ?

File: cfme/infrastructure/host.py (hunk 1 above)
Comment (quarckster) [inline cfme/infrastructure/host.py]: I think `execute_button` button should only click on the custom button. Some dialog can be opened and there will no any flash messages.

Comment (izapolsk) [inline cfme/infrastructure/host.py]: okay, but please check maybe this test needs checking flash message

**Comment by quarckster** [comment]
@izapolsk  PRT failed for some reason which is not related to this PR. I can fix it, but it will be another PR.

**Comment by izapolsk** [comment]
@quarckster, the problem is that you changes cannot be tested then. Have you ever tested them locally ?

File: cfme/infrastructure/host.py (hunk 1 above)
Comment (quarckster) [inline cfme/infrastructure/host.py]: This method is used in only one test which is ["test_button_on_host"](https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/automate/test_buttons.py#L133). There are no any flash message checks.

**Comment by quarckster** [comment]
@izapolsk yes, I've tested my changes locally. Custom button is clicked.

**Pull request merged**